echoregions/lines/lines.py

```diff
@@ -179,12 +163,14 @@ def plot(
                 f"start and end times are of type {type(start_time)} and {type(end_time)}. \
                             They must be of of type Pandas Timestamp."
             )
```

Comment:
better to check each of these separately so the error message can be more specific.